hack/push-updates.sh

```diff
@@ -110,28 +131,20 @@ REMOTE_NAME="working"
 
 set -e
 
-function fork_repo () {
-    local repo_name="$1"
-
-    if ! hub fork --remote-name $REMOTE_NAME; then
-        # Forking failed, so maybe we have another repo with that name
-        # forked from a different source. Try adding that as a remote.
-        echo "Forking failed, trying a simple clone"
-        local github_user=$(git config --get github.user)
-        git remote add $REMOTE_NAME git@github.com:${github_user}/${repo_name}.git
-        git remote update $REMOTE_NAME
-    fi
-}
-
 for repo in $REPOS; do
     echo
     echo "Building PR for $repo"
     echo
 
-    hub clone openshift/$repo
+    gh repo clone openshift/$repo
     pushd ./$repo
 
-    git checkout origin/$BRANCH_NAME
+    if [ -z "$BRANCH_NAME" ]; then
+      git checkout
+    else
+      git checkout origin/$BRANCH_NAME
+    fi
+
```

Comment:
I think we should avoid using the `$BRANCH_NAME` here. As we effectively use this as the MAO branch we want to reference.
In these provider repositories the default branch varies and this specific one won't work for them.

I'd just drop this branched checkout entirely as we should be able to rely on the fact that the branch already checked out when we clone the repo, should be the default one. Or, worst case scenario we can use some heuristic to find that out: https://stackoverflow.com/questions/28666357/git-how-to-get-default-branch

Reply:
Ok I see, makes sense then 👍